src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
+
+    get coords() {
+      return {
+        x: this.x,
+        y: this.y,
+      };
+    },
```

Comment:
```suggestion
    coords: {
      x: 0,
      y: 0,
    },
```
Read the task description carefully.
`The coordinates of the robot must be stored in the object coords inside the robot.`